### ISSUE TITLE
hello, this method has a heavy security issue.

### ISSUE THREAD

**Issue by stamhe** [enteropositivo/bip39colors#2]
https://github.com/enteropositivo/bip39colors/tree/main/manual

as this document, 
after step 4, for keeping mnemonic phrase order , add a number [00-23]  as the prefix.
after step 5,  convert the digit number to hex string, [00-23] has the [solid hex string]. for example, [12] => #B, [14] => #E, they are solid string, when computer has  beed hacked, the hacker could scan solid string "#B", "#E".

so this method may be needed more optimized.

**Comment by enteropositivo** [comment]
No,  the method never accepts [00-23]  as prefix, only  [00-15]  is allowed,  this way only valid hex color codes are generated.

**Comment by stamhe** [comment]
if only [00-15] is allowed, it's the same.

[12] => #B, [14] => #E 

"#B", "#E" are the special solid string.
